tiled/client/container.py

```diff
@@ -382,11 +382,44 @@ def __getitem__(self, keys, _ignore_inlined_contents=False):
             )
         return result
 
-    def delete(self, key):
+    def delete_contents(
+        self,
+        keys: Optional[Union[str, Iterable[str]]] = None,
+        recursive: bool = False,
+        external_only: bool = True,
+    ) -> "Container":
+        """Delete the contents of this Container.
+
+        Parameters
+        ----------
+        keys : str or list of str
+            The key(s) to delete. If a list, all keys in the list will be deleted.
+            If None (default), delete all contents.
+        recursive : bool, optional
+            If True, descend into sub-containers and delete their contents too.
+            Defaults to False.
+        external_only : bool, optional
+            If True, only delete externally-managed data keys. Defaults to True.
```

Comment:
```suggestion
            If True, only delete externally-managed data. Defaults to True.
```

Avoiding "data keys" as an overloaded term (bluesky). And the sentence works fine with just "data".